js/viz/docs/doccharts.js

```diff
@@ -4111,7 +4111,7 @@ var BaseChart = {
     /**
     * @name basechartmethods_getallseries
     * @publicName getAllSeries()
-    * @return array
+    * @return Array<Object>
```

Comment:
This notation will lead to problems with usage ( for example: you can't write getAllSeries()[0].color. You should cast to 'series' type or 'any' ). I recommend you use 'any' type Instead 'Object'. In the future we will reorganize our definitions for more better usage.

Reply:
I have changed definition to Array<baseSeriesObject>